### PR TITLE
CB-260: Show external urls and related places on place pages.

### PR DESCRIPTION
**Pull request by ferbncode**
Added code for showing `external urls` on place pages and related places (only if places are a `part of` another place). Also, added `lastfm-16.png` for related `last.fm` urls.

### CODE REVIEW

**Comment by ferbncode** [comment]
Some screenshots are:

external-links
places-partof

**Comment by gentlecat** [comment]
Can you rebase this please? I merged the underlying DB access changes.

File: critiquebrainz/frontend/templates/place/entity.html [hunk 1]
```diff
@@ -92,6 +92,35 @@ <h4 style="margin-bottom:0;">{{ _('Reviews') }}</h4>
...
+        <b>{{ _('External links') }}</b>
+          <ul class="list-unstyled external-links">
+            {% for url in place['external-urls'] %}
+              <li class="clearfix">
+                <div class="favicon-container">
+                  {% if url.icon %}
+                    <img src="{{ get_static_path('favicons/'+url.icon) }}" />
+                  {% else %}
+                    <img src="{{ get_static_path('favicons/external-16.png') }}" />
+                  {%endif%}
```
Comment (gentlecat) [inline critiquebrainz/frontend/templates/place/entity.html]: `{% endif %}`

Comment (ferbncode) [inline critiquebrainz/frontend/templates/place/entity.html]: 😅

**Comment by alastair** [comment]
this looks really cool!

**Comment by ferbncode** [comment]
Thanks :smile:

**Pull request merged**